backends/tc/backend.cpp

```diff
@@ -354,6 +354,32 @@ void ConvertToBackendIR::updateDefaultMissAction(const IR::P4Table *t, IR::TCTab
                 if (defaultActionProperty->isConstant) {
                     tabledef->setDefaultMissConst(true);
                 }
+                bool directionParamPresent = false;
+                auto paramList = actionCall->action->getParameters();
+                for (auto param : paramList->parameters) {
+                    if (param->direction != IR::Direction::None) directionParamPresent = true;
```

Comment:
All parameters should be directionless?

Reply:
For now, default action with directionless parameter is handled
For direction parameter, they are still discussing, hence we need not set any default param value until then